src/objects/form-group/form-group.stories.mdx

```diff
@@ -1,35 +1,50 @@
 import { Canvas, Meta, Story } from '@storybook/addon-docs';
-import demo from './demo/input.twig';
+import basicDemo from './demo/input.twig';
+import badgeDemo from './demo/badge.twig';
+// The '!!raw-loader!' syntax is a non-standard, Webpack-specific, syntax.
+// See: https://github.com/webpack-contrib/raw-loader#examples
+// For now, it seems likely Storybook is pretty tied to Webpack, therefore, we
+// are okay with the following Webpack-specific raw loader syntax. It's better
+// to leave the ESLint rule enabled globally, and only thoughtfully disable as
+// needed (e.g. within a Storybook docs page and not within an actual
+// component). This can be revisited in the future if Storybook no longer relies
+// on Webpack.
+// eslint-disable-next-line @cloudfour/import/no-webpack-loader-syntax
+import basicDemoSource from '!!raw-loader!./demo/input.twig';
+// eslint-disable-next-line @cloudfour/import/no-webpack-loader-syntax
+import badgeDemoSource from '!!raw-loader!./demo/badge.twig';
```

Comment:
I just couldn't keep copying and pasting the revised source over, so carrying this dirty hack from other stories for now.